src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/SVAnnotateEngine.java

```diff
@@ -0,0 +1,734 @@
+        }
+
+        return intervals;
+    }
+
+
+
+    /**
+     * Create a copy of the variant consequence dictionary in which the feature names for each consequence are sorted
+     * in alphabetical order
+     * @param variantConsequenceDict - running map of consequence -> feature name for variant to update
+     * @return - a consequence -> feature name map in which the feature names are sorted alphabetically
+     */
+    @VisibleForTesting
+    protected static Map<String, Object> sortVariantConsequenceDict(final Map<String,Set<String>> variantConsequenceDict) {
+        final Map<String, Object> sorted = new HashMap<>();
+        for (final String consequence : variantConsequenceDict.keySet()) {
+            final List<String> sortedGenes = new ArrayList<>(variantConsequenceDict.get(consequence));
+            Collections.sort(sortedGenes);
+            sorted.put(consequence, sortedGenes);
+        }
+        return sorted;
+    }
+
+    /**
+     * Create a consequence -> feature name map and add all annotations for protein-coding, promoter, nearest TSS,
+     * and noncoding consequences for a variant
+     * @param variant - VCF record
+     * @param gtfIntervalTree - interval tree of protein-coding transcripts from GTF to find overlappers with variant
+     * @param promoterIntervalTree - interval tree of promoters from GTF to find promoters overlapping variant
+     * @param transcriptionStartSiteTree - interval tree of TSS locations from GTF to find nearest TSS to variant
+     * @param nonCodingIntervalTree - interval tree of noncoding elements from BED to find features overlapping variant
+     * @param MSVExonOverlapClassifications - consequence classes to reclassify for CNVs
+     * @param sequenceDictionary - SAMSequenceDictionary from VCF
+     * @param maxBreakendLen - Max size of BND in bp to annotate as DEL / DUP if applicable
+     * @return - map of consequence -> feature name containing all annotations for the variant
+     */
+    @VisibleForTesting
+    protected static Map<String, Object> annotateStructuralVariant(final VariantContext variant,
+                                                                   final SVIntervalTree<GencodeGtfTranscriptFeature> gtfIntervalTree,
+                                                                   final SVIntervalTree<String> promoterIntervalTree,
+                                                                   final SVIntervalTree<String> transcriptionStartSiteTree,
+                                                                   final SVIntervalTree<String> nonCodingIntervalTree,
+                                                                   final Set<String> MSVExonOverlapClassifications,
+                                                                   final SAMSequenceDictionary sequenceDictionary,
+                                                                   final int maxBreakendLen) {
```

Comment:
```suggestion
    protected Map<String, Object> annotateStructuralVariant(final VariantContext variant) {
```
Fewer lines. IMO, I wouldn't use static if most of the arguments are class members.